src/averageAges.js

```diff
@@ -20,6 +20,16 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const filterMan = !century
+    ? people.filter(human => human.sex === 'm')
+    : people.filter(human =>
+      Math.ceil(human.died / 100) === century && human.sex === 'm');
+
+  const menAge = filterMan.map(human => human.died - human.born);
+
+  const averageAge = menAge.reduce((a, b) => a + b) / menAge.length;
+
+  return averageAge;
```

Comment:
You've used this logic in every function, let's move it to separate helper function